demos/3d_segmentation_demo/python/README.md

```diff
@@ -56,7 +56,7 @@ Options:
 ```
 
 Running the application with the empty list of options yields the usage message and an error message.
-To run the demo, use public or pre-trained models that support 3D convolution, for example, UNet3D. You can download the pre-trained models using the OpenVINO [Model Downloader](../../../tools/downloader/README.md). The list of models supported by the demo is in [models.lst](./models.lst).
+To run the demo, use public or pre-trained models that support 3D convolution, for example, UNet3D. You can download the pre-trained models using the OpenVINO [Model Downloader](../../../tools/downloader/README.md). The list of models supported by the demo is in the `models.lst` file in the demo's directory.
```

Comment:
```suggestion
To run the demo, use public or pre-trained models that support 3D convolution, for example, UNet3D. You can download the pre-trained models using the OpenVINO [Model Downloader](../../../tools/downloader/README.md). The list of models supported by the demo is in `<omz_dir>/demos/3d_segmentation_demo/python/models.lst`.
```

This would be more direct, as well as more robust, because I plan to implement automatic checking for `<omz_dir>` references.